Jenkinsfile

```diff
@@ -16,7 +16,7 @@ properties([
     string(name: 'VERSION',
            defaultValue: '',
            description: 'Override automatic versioning'),
-    choice(choices: ["run", "skip"].join("\n"),
```

Comment:
It doesn't look like there is a robotest step in this pipeline anymore.  Is this (and the other robotest paramters) needed at all?

Reply:
Because jenkinsfile is for M customer only I just deleted it for now. I will add it back if needed later.